primitives/application-crypto/src/sr25519.rs

```diff
@@ -19,16 +19,35 @@
 use crate::{RuntimePublic, KeyTypeId};
 
 use sp_std::vec::Vec;
+use sp_core::crypto::CryptoTypeId;
 
 pub use sp_core::sr25519::*;
 
+/// An identifier used to match public keys against sr25519 keys
+pub const CRYPTO_ID: CryptoTypeId = CryptoTypeId(*b"sr25");
```

Comment:
Why is it defined for the second time here? Can't we use the same thing that is in `sp_core`?